website/docs/components/table/advanced-table/partials/guidelines/guidelines.md

```diff
@@ -236,17 +229,16 @@ Striping enhances readability by alternating row colors, making it easier to sca
 The row placement property is only relevant within Figma and doesn’t exist as a property within the code.
 !!!
 
-Row placement determines the visual styling based on where the row is placed relative to other rows within the Advanced Table. Only cells with a column placement that is either start or end use the row placement property; column position middle does not use this property.
+Row placement determines the visual styling based on where the row is placed. Only cells with a column placement that is either `start` or `end` use the row placement property; column position `middle` does not use this property.
 
 ![The cell with column placement end and row placement end has a border radius set on the bottom right corner.](/assets/components/table/advanced-table/table-row-placement.png)
 
 ## Cells
```

Comment:
[suggestion] There's additional content under the Cells header in the Table docs that feels particularly helpful. Should we consider adding it to the Advanced Table docs as well?

> For the user to scan, sort, and filter the table easily, each cell should contain a single piece of data. Having more than one piece of data in a cell makes it harder for users to navigate the relationships between headers and cells.